tests/tests/tier0/bluechictl-metrics/python/bluechictl_metrics.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import re
+import time
+import unittest
+
+from bluechi_machine_lib.bluechictl import BackgroundRunner, BluechiCtl, SimpleEventEvaluator
+
+node_foo_name = "node-foo"
+service_simple = "simple.service"
+
+
+class TestBluechictlMetrics(unittest.TestCase):
+
+    def setUp(self) -> None:
+        signal_agent_metrics_start_pat = re.compile(r"^.*Agent systemd StartUnit job.*")
+        signal_agent_metrics_stop_pat = re.compile(r"^.*Agent systemd StopUnit job.*")
+        signal_controller_metrics_pat = re.compile(r"^.*BlueChi job gross measured time.*")
+        expected_patterns = {signal_agent_metrics_start_pat,
+                             signal_agent_metrics_stop_pat,
+                             signal_controller_metrics_pat}
+        args = ["metrics", "listen"]
+        self.bgrunner = BackgroundRunner(args, SimpleEventEvaluator(expected_patterns))
+        self.bluechictl = BluechiCtl()
+
+    def test_bluechictl_metrics(self):
+        self.bluechictl.metrics_enable()
+        self.bgrunner.start()
+        time.sleep(0.5)
+        self.bluechictl.unit_start(node_foo_name, service_simple)
+        time.sleep(0.5)
```

Comment:
Do we need the timeout between start and stop events?

Reply:
What do you mean? You're not talking about `time sleep`s

Reply:
Removed